releasenotes/notes/go-language-detection-90f3908616e188e0.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds Go language and version detection through the system probe. This beta feature can be enabled by adding
+    `language_detection: { enabled: true }` to your `system-probe.yaml`.
```

Comment:
I don't think it's standard to use JSON syntax in YAML even though it's syntactically valid - our sample configs don't seem to have any examples of this: https://github.com/DataDog/datadog-agent/blob/1183cd387142b79b6f649365db1599e9e14659c5/pkg/config/config_template.yaml

Reply:
Hmm fair enough. I can change the wording here to reflect that.